testsuite/src/commonMain/kotlin/org/jetbrains/litmuskt/tests/UnsafePublication.kt

```diff
@@ -66,19 +66,42 @@ object UnsafePublication {
         }
     }
 
-    val PlainArray = litmusTest({
+    val PlainArrayBoxed = litmusTest({
```

Comment:
Maybe leave the name `PlainArray` (for consistency with `PlainIntArray`, since it uses `IntArray`, and this test used `Array` class)?